nextcloud/richdocuments
# Document targets

- Implement endpoint to open files in the browser (replaces in #2419)
  - In addition to the linked PR: Add endpoint to open Office documents at the provided target
- feat: Implement reference picker for document sections
  - Add to the smart picker
  - Implement widget to render link to document sections

For now limited to documents, spreadsheets and presentations have pending fixes on the collabora side.

Steps to test:
- Upload a file with some sections
[example.odt](https://github.com/nextcloud/richdocuments/files/11611142/example.odt)
- Have collabora setup
- E.g. in text: Open the smart picker and insert a link to a office document target
- See the preview and click the link to jump to the section directly
